### DISCUSSION TITLE
Question over tags in query column of Server timings tab

### DISCUSSION THREAD

**Discussion by InterestedUser1** [DaxStudio/DaxStudio#1085] [General]
Hi All, 
I have a question, not sure if its an "issue" but why am I seeing <ccon> </ccon> tags all over?
Thanks in advance!

![image](https://user-images.githubusercontent.com/128793112/227470750-0536069d-1738-4c79-b3ff-f27358a2b319.png)

**Comment by marcosqlbi** [comment]
Are you connected to Power BI Service through XMLA endpoint or Azure Analysis Services?

**Reply by InterestedUser1** [reply]
Thanks for the blazing fast response Marco! 

Yes I am connected to t he service via XMLA endpoint.

**Comment by marcosqlbi** [comment]
Make sure you use the latest version of DAX Studio with this setting enabled:
![image](https://user-images.githubusercontent.com/13840816/227472568-f9cb000f-0b8e-44c9-b79a-5d3a95c85d5f.png)
Let us know if you still have the issue after this check.

**Comment by InterestedUser1** [comment]
Hi Marco, 
The update did the trick! I was still running v2.17 with the Simplify option ENABLED. 
So updating to latest version of Dax Studio worked. Thanks again for your quick response! 

P.S. Im a big fan!

**Comment by marcosqlbi** [comment]
Thanks for closing the loop!
